pipeline_documentation.md

```diff
@@ -370,18 +370,21 @@ by feature with a [**custom script**][custom-script-mir-ext].
 
 > Adjust miRNAs' 'Name' attribute to account for the different genomic
 > locations the miRNA sequence is annotated on and ensure their uniqueness.
-> The name format is `SPECIES-mir-NAME-#` for pri-miRs, and 
+> The name format is `SPECIES-mir-NAME-#` for pri-miRs, and
 > `SPECIES-miR-NAME-#-ARM` or `SPECIES-miR-NAME-#` for mature miRNA with both
 > or just one arm respectively, where `#` is the replica integer. If a pri-miR
 > has a replica but its number is set in the 'ID' attribute, the first instance
-> does not has a suffix but the other one(s) do. If a precursor has no other
-> occurrences, no further modifications are made. On the other hand,
-> mature miRNA regions are extended on both sides to account for isomiR species
-> with shifted start and/or end positions without exceeding chromosome(s)
-> boundaries. If required, pri-miR loci are also extended to accommodate the
-> new miRNA coordinates. In addition, pri-miR names are modified to record the
-> final positions by appending `_-y` and `_+x` to them, where `y` is the 5'
-> shift and `x` the 3' shift.
+> does not has a suffix but the other one(s) do. In addition, the
+> 'Derives_from' attribute is updated to match the precursor 'ID' attribute.
+> If a precursor has no other occurrences, but its `NAME` portion of the
+> 'Name' attribute is different from the `NAME` of its corresponding mature
+> miRNA(s), then the mature miRNA(s) `NAME` is updated to match the precursor
+> one. On the other hand, mature miRNA regions are extended on both sides to
+> account for isomiR species with shifted start and/or end positions without
+> exceeding chromosome(s) boundaries. If required, pri-miR loci are also
+> extended to accommodate the new miRNA coordinates. In addition, pri-miR names
+> are modified to record the final positions by appending `_-y` and `_+x` to
+> them, where `y` is the 5' shift and `x` the 3' shift.
```

Comment:
Oh dear :see_no_evil:

It's quite complicated, isn't it? Not sure if there's anything we can (or want to) do about the actual complexity, but perhaps run it through some GenAI to at least get the wording as simple as possible.